docs/source/Htypes.rst

```diff
@@ -287,6 +287,69 @@ array([[965, 110, 262,  77],
 >>> ds.boxes.append(boxes)
 
 
+.. _bbox-3d-htype:
+
+Bounding Box Htype
+~~~~~~~~~~~~~~~~~~
+
+- :bluebold:`Sample dimensions:` ``(# bounding boxes, 9)``
+
+Bounding boxes are in KITTI format. In order for bounding boxes to be correctly displayed by the visualizer, the format of the bounding box should be as follows:
+
+``[x, y, z, length, width, hieght, rot_x, -rot_z, rot_y]``
```

Comment:
why does rot_z have a minus symbol? Also follow this same ordering in the descriptions below